src/util/common.ts

```diff
@@ -6,6 +6,9 @@ import { EventEmitter } from 'events';
  */
 export class Common {
 
+  public static DEFAULT_TIMEOUT: number =  2000;
+  public static LONG_TIMEOUT: number =  6000;
```

Comment:
Please mark these readonly so they are truly constant.
Also, properties are public by default, no need to use the explicit modifier.

Reply:
corrected.

Reply:
because one should in tests assert what is significant for the function/outcome: the request object which is API. Asserting how the code is organized just adds clutter, it has no significance for the function. As one finds ex. https://howtodoinjava.com/best-practices/unit-testing-best-practices-junit-reference-guide/#2:
> 2. Don’t make unnecessary assertions
> 
> Remember, unit tests are a design specification of how a certain behavior should work, 
> not a list of observations of everything the code happens to do.